qiskit/transpiler/passes/__init__.py

```diff
@@ -133,7 +133,7 @@
    DAGLongestPath
 
 Synthesis
-=============
+=========
```

Comment:
We should add thew transpiler pass to this list and re-export it from this module so people can do `from qiskit.transpiler.passes import SolovayKitaev`